### ISSUE TITLE
Training using Supcon loss is slower than cross_entropy loss

### ISSUE THREAD

**Issue by yinbing668** [HobbitLong/SupContrast#151]
I find that in the same case(same dataloader, same network architecture), training using Supcon loss, the accuracy improves slower than the cross_entropy loss. Do you find the same case?
Besides, i cannot run the code, it report"segment error".

**Comment by tian1327** [comment]
check this ICML'21 paper: https://arxiv.org/abs/2102.08817
